conans/model/profile.py

```diff
@@ -24,6 +24,7 @@ def __init__(self):
         self.conf = ConfDefinition()
         self.buildenv = ProfileEnvironment()
         self.runenv = ProfileEnvironment()
+        self.runner = {}
```

Comment:
It seems the ``Profile`` might need to add ``serialize()`` and ``dumps()`` for full management of the ``runners`` information